lib/matplotlib/mathtext.py

```diff
@@ -40,10 +40,12 @@
                                        tex2uni, latex_to_cmex,
                                        stix_virtual_fonts)
 
+_log = logging.getLogger(__name__)
 
 ##############################################################################
 # FONTS
 
+
```

Comment:
no empty line here?

Reply:
I added it because I got ```E302 expected 2 blank lines, found 1 ``` from flake8 when it wasn't there.
Is there a better solution?

Reply:
Putting the blank line *before* the ##### seems more logical?  Should hopefully also placate the style checker.

Reply:
You're right. That fixed it.